dll/ntdll/ldr/ldrapi.c

```diff
@@ -982,18 +984,19 @@ LdrVerifyImageMatchesChecksum(IN HANDLE FileHandle,
 
 NTSTATUS
 NTAPI
-LdrQueryProcessModuleInformationEx(IN ULONG ProcessId,
-                                   IN ULONG Reserved,
-                                   OUT PRTL_PROCESS_MODULES ModuleInformation,
-                                   IN ULONG Size,
-                                   OUT PULONG ReturnedSize OPTIONAL)
+LdrQueryProcessModuleInformationEx(
+    _In_opt_ ULONG ProcessId,
+    _Reserved_ ULONG Reserved,
+    _Out_writes_bytes_to_(Size, *ReturnedSize) PRTL_PROCESS_MODULES ModuleInformation,
+    _In_ ULONG Size,
+    _Out_opt_ PULONG ReturnedSize)
 {
     PLIST_ENTRY ModuleListHead, InitListHead;
     PLIST_ENTRY Entry, InitEntry;
     PLDR_DATA_TABLE_ENTRY Module, InitModule;
     PRTL_PROCESS_MODULE_INFORMATION ModulePtr = NULL;
     NTSTATUS Status = STATUS_SUCCESS;
```

Comment:
https://github.com/reactos/reactos/blob/82241f85e4508ffe02bdb9538b570fc977069c59/dll/ntdll/ldr/ldrapi.c#L1017
is redundant.